scripts/generateAssetData/ethereum/uniswapV2Pools.ts

```diff
@@ -1,25 +1,156 @@
-import { ethChainId as chainId, toAssetId } from '@shapeshiftoss/caip'
-import type { Asset } from '@shapeshiftoss/types'
-
-import { ethereum } from '../baseAssets'
-import { colorMap } from '../colorMap'
-
-export const getUniswapV2Pools = (): Asset[] => {
-  const assetNamespace = 'erc20'
-  const assetReference = '0x470e8de2ebaef52014a47cb5e6af86884947f08c' // Uniswap V2 - FOX/WETH contract address
-  const assetId = toAssetId({ chainId, assetNamespace, assetReference })
-  const foxWeth: Asset = {
-    assetId,
-    chainId,
-    name: 'ETH/FOX Pool',
-    precision: 18,
-    color: colorMap[assetId] ?? '#FFFFFF',
-    icon: 'https://assets.coincap.io/assets/icons/256/uni.png',
-    symbol: 'UNI-V2',
-    explorer: ethereum.explorer,
-    explorerAddressLink: ethereum.explorerAddressLink,
-    explorerTxLink: ethereum.explorerTxLink,
+import { type AssetId, ethAssetId, ethChainId } from '@shapeshiftoss/caip'
+import { fromAssetId, toAssetId } from '@shapeshiftoss/caip/src/assetId/assetId'
+import type { Asset, AssetsByIdPartial, PartialRecord } from '@shapeshiftoss/types'
+import type { AxiosRequestConfig } from 'axios'
+import axios from 'axios'
+// @ts-ignore we don't have typings for this bad boi
+import crypto from 'crypto-browserify'
+import qs from 'qs'
+
+import { WETH_TOKEN_CONTRACT_ADDRESS } from '../../../src/contracts/constants'
+import generatedAssetData from '../../../src/lib/asset-service/service/generatedAssetData.json'
+import {
+  chainIdToZapperNetwork,
+  V2AppTokensResponse,
+  ZapperGroupId,
+  zapperNetworkToChainId,
+} from '../../../src/state/apis/zapper/validators'
+
+const sha256 = (input: string): string => crypto.createHash('sha256').update(input).digest('hex')
```

Comment:
A bunch of reimports here because trying to import from their original source will cause a bunch of errors from our TS paths